source/OptimaJet.DWKit.StarterApplication/Services/HttpOrganizationContext.cs

```diff
@@ -11,7 +11,7 @@ public class HttpOrganizationContext : IOrganizationContext
         public HttpContext HttpContext { get; set; }
         public OrganizationService OrganizationService { get; set; }
 
-        private bool parsed;
+        private bool parsed = false;
```

Comment:
why was this change made?

Reply:
It was changed as part of a debug session and accidentally checked in.  I've put it back.  Was really weird.  I had put a breakpoint in Parse, and it always came up true and would never execute the method.  The only way to make it work was to remove the breakpoint.  Anyway, didn't mean that to get checked in.

Reply:
sounds like some race condition could be present or something